tests/unit/dojson/test_dojson_hep.py

```diff
@@ -962,18 +975,25 @@ def test_authors_from_100__a_j_m_u_v_w_y():
                     'value': 'D.Macnair.2',
                 },
             ],
+            'raw_affiliations': [
+                {
+                    'value': 'SLAC, Menlo Park, California, USA'
+                },
+            ],
         },
     ]
     result = hep.do(create_record(snippet))
 
     assert expected == result['authors']
 
 
-def test_authors_from_100__a_u_x_w_y_z_with_malformed_x():
+def test_authors_from_100__a_u_v_x_w_y_z_with_malformed_x():
     snippet = (
         '<datafield tag="100" ind1=" " ind2=" ">'
         '  <subfield code="a">Bakhrushin, Iu.P.</subfield>'
         '  <subfield code="u">NIIEFA, St. Petersburg</subfield>'
+        '  <subfield code="v">SLAC, Menlo Park, California, USA</subfield>'
+        '  <subfield code="v">SLAC, Menlo Park, IT</subfield>'
```

Comment:
The record that generated this test does not have this, so you can't add it here.

These unit tests document **regressions**, that is, bugs that have since been fixed. You don't modify those, because you risk reintroducing the bug.

Find a record that does actually have a `100__v`, and create a new test.